packrat/src/test/java/com/github/reflectoring/infiniboard/packrat/source/WidgetConfigTest.java

```diff
@@ -25,22 +25,27 @@
 @Category(MongoIntegrationTests.class)
 public class WidgetConfigTest {
 
-    private static final String TITLE          = "myFirstWidget";
+    private static final String WIDGETTITLE    = "myFirstWidget";
+    private static final String WIDGETTYPE     = "myFirstWidgetType";
```

Comment:
Please use valid data when testing. Also it would work with this string, we should prefer close to production test data.